win32ss/drivers/videoprt/resource.c

```diff
@@ -208,15 +208,16 @@ IntVideoPortReleaseResources(
     BOOLEAN ConflictDetected;
     // An empty CM_RESOURCE_LIST
     UCHAR EmptyResourceList[FIELD_OFFSET(CM_RESOURCE_LIST, List)] = {0};
-
-    Status = IoReportResourceForDetection(
-                DeviceExtension->DriverObject,
-                NULL, 0, /* Driver List */
-                DeviceExtension->PhysicalDeviceObject,
-                (PCM_RESOURCE_LIST)EmptyResourceList,
-                sizeof(EmptyResourceList),
-                &ConflictDetected);
-
+    if (DeviceExtension->IsLegacyDevice)
```

Comment:
```suggestion

    if (DeviceExtension->IsLegacyDevice)
```
(newline separating variables from code)